src/rgw/rgw_main.cc

```diff
@@ -691,6 +691,7 @@ static int process_request(RGWRados *store, RGWREST *rest, RGWRequest *req, RGWC
   if (handler)
     handler->put_op(op);
   rest->put_handler(handler);
+  dout(1) << "API logging: " << s->trans_id.c_str() << "        " << http_ret <<dendl;
```

Comment:
Make this message more clear. Like in the next line it explains what http_ret means. (http status). Also why is this needed when the next line prints this?